src/index.js

```diff
@@ -28,9 +28,13 @@ export default class Fetch {
     return new Fetch(defaultURL, defaultHeaders, options)
   }
 
-  buidAuthorizationHeader(key) {
+  buildAuthorizationHeader(key) {
     if (key) {
-      const authKeyword = this.options.defaultAuthorizationKeyword || DEFAULT_AUTHORIZATION_KEYWORD
+      const authKeyword = (
```

Comment:
We are talking about having an empty header name, like:
```
Content-type: something
Accept: */*
: somevalue <<< this
Content-length: 2342432
```

So, either I'm getting this wrong or it doesn't make any sense.